schutzbot/Jenkinsfile

```diff
@@ -133,6 +133,7 @@ pipeline {
                         AZURE_CREDS = credentials('azure')
                         OPENSTACK_CREDS = credentials("psi-openstack-creds")
                         VCENTER_CREDS = credentials('vmware-vcenter-credentials')
+                        DISTRO_CODE = "fedora31"
```

Comment:
I think you can reuse `ID` and `VERSION_ID`. See https://github.com/osbuild/osbuild-composer/blob/85423d403c031f9931568bb2525bcb2d4dd7eb94/test/image-tests/aws.sh#L7

Reply:
I was afraid that it would clash for RHEL 8 in case we test against beta and nightly channels. This seemed to be a simple and safe workaround, but if you think there is nothing to worry about I can change it for sure.

Reply:
Right, let's merge this and worry about this later if we want to.